src/modules/mealDisplay.js

```diff
@@ -0,0 +1,57 @@
+import likesListener from './likes.js';
+import displayPopup from './display-popup-comment.js';
+import { handleSaveComment } from './display-comments.js';
+
+const mealList = document.getElementById('list-home');
+
+export default function renderRecipes(recipe, appId) {
+  const li = document.createElement('li');
+  li.classList.add('card');
+
```

Comment:
- [ ] As suggested by previous reviewer please use arrow functions instead of function keyword as its been depreciated.